package.json

```diff
@@ -8,7 +8,7 @@
   "react-native": "src/index",
   "source": "src/index",
   "expoVersion": "",
-  "cioNativeiOSSdkVersion": "= 2.9.2",
+  "cioNativeiOSSdkVersion": "= 2.11.0",
```

Comment:
Would appreciate it if you could run `pod update` and run the sample apps before merging once iOS changes are live. This will ensure our `main` branch has the updated state for both the SDK and the sample apps.

Reply:
👍 to this